devops/README.md

```diff
@@ -0,0 +1,14 @@
+# multinet-server/devops
+
+The `arangodb.yml` Ansible playbook in this directory installs and configures
+ArangoDB on an Ubuntu server. It does the following:
+- installs ArangoDB
+- sets a default password (TODO: make this password configurable)
```

Comment:
Can we make this an issue instead of leaving it in the README?

Reply:
Good idea. Check out 0662094.